packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/regression-4348/expected.js

```diff
@@ -0,0 +1,6 @@
+function first() {
+  var _ref;
+
+  var index = 0;
+  return _ref = index++ + 0, arguments.length <= _ref ? undefined : arguments[_ref];
```

Comment:
It's the index of the rest parameter. If the function signature was e.g. `function(a, b, ...values)`, it would be `index++ + 2` instead.